x-pack/plugins/security_solution_serverless/public/upselling/register_upsellings.tsx

```diff
@@ -93,105 +62,20 @@ export const registerUpsellings = (
     {}
   );
 
-  upselling.setPages(upsellingPagesToRegister);
-  upselling.setSections(upsellingSectionsToRegister);
-  upselling.setMessages(upsellingMessagesToRegister);
-};
-
-// Upselling for entire pages, linked to a SecurityPageName
-export const upsellingPages: UpsellingPages = [
```

Comment:
This file was becoming too crowded and with mixing concepts. I moved the upselling definitions to 
[x-pack/plugins/security_solution_serverless/public/upselling/upsellings.tsx](https://github.com/elastic/kibana/pull/187105/files#diff-6b0c97dc44ef906d7c7acc0544f39a7b956e391ef60390119c634e090f520ac9)
keeping only the registration logic here.